### DISCUSSION TITLE
Crashing when not in full screen

### DISCUSSION THREAD

**Discussion by eddy-99** [sn4k3/UVtools#316] [General]
Just a heads up, I unchecked start maximized and when I performed an operation in half screen program kept crashing. Easy fix start in full screen.

Eddy

**Comment by sn4k3** [comment]
You need to elaborate more what you are experiencing, and if possible with screenshots, so others can understand the issue and able to help.
Uncheck Start Maximized, run program and run a operation does not crash the program for me

**Comment by eddy-99** [comment]
Yesterday program worked faultless. I unchecked start maximized.
Opened program today in half screen, loaded file and run issues, a list of issues came up as should.
Clicked on attempt to repair, new window flashed but program froze.
In the taskbar 2 tabs were open program and repair issues tabs but unable to restore the tabs.
![Capture](https://user-images.githubusercontent.com/57560700/136585805-27611058-ebb6-4347-965c-7911f5d2c575.PNG)
m.

**Reply by sn4k3** [reply]
Ok froze is diferent from crashing...
I was able to replicate that, the window is open but invisible, still it accept keys like Enter or Esc to accept or close the dialog. Dialog windows waits util that dialog closes and lock the other background UIs, giving the impression that the program is frozen, but it just waiting on dialog response.
This is related to the UI framework which have some bugs and out of my scope
An easy fix is to resize window just by any pixel and it will work, but you will have to do this per UVtools instance which can be annoying

**Reply by eddy-99** [reply]
Thanks, not a problem as I will keep it full screen.
